pkg/build/graph/analysis/bc.go

```diff
@@ -238,6 +238,16 @@ func FindPendingTags(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// getImageStreamTagSuggestion will return the appropriate marker Suggestion for when a BuildConfig is missing is input ImageStreamTag;  in particular,
+// it will determine whether or not another BuildConfig can produce the aforementioned ImageStreamTag
+func getImageStreamTagSuggestion(g osgraph.Graph, f osgraph.Namer, tagNode *imagegraph.ImageStreamTagNode, bcNode *buildgraph.BuildConfigNode) osgraph.Suggestion {
+	if _, points := buildPointsToTag(g, tagNode); points {
+		return osgraph.Suggestion(fmt.Sprintf("%s needs to be imported or created by a build of %s.", f.ResourceName(tagNode), f.ResourceName(bcNode)))
```

Comment:
also it seems like the suggested text is going to tell the user to run the build that had the problem in the first place, rather than to run the build that would produce the needed imagestreamtag?